### ISSUE TITLE
Release new version?

### ISSUE THREAD

**Issue by basnijholt** [abcminiuser/python-elgato-streamdeck#118]
Hi @abcminiuser,

Currently on my Macbook M2 I have to use latest `main` in order to get my StreamDeck to connect.

Otherwise, I run into:
```
  File "/Users/basnijholt/micromamba/envs/streamdeck-yaml/bin/home-assistant-streamdeck-yaml", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/Users/basnijholt/Sync/Overig/home-assistant/home-assistant-streamdeck-yaml/home_assistant_streamdeck_yaml.py", line 1925, in main
    asyncio.run(
  File "/Users/basnijholt/micromamba/envs/streamdeck-yaml/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/Users/basnijholt/micromamba/envs/streamdeck-yaml/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/basnijholt/micromamba/envs/streamdeck-yaml/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/Users/basnijholt/Sync/Overig/home-assistant/home-assistant-streamdeck-yaml/home_assistant_streamdeck_yaml.py", line 1832, in run
    deck = get_deck()
           ^^^^^^^^^^
  File "/Users/basnijholt/Sync/Overig/home-assistant/home-assistant-streamdeck-yaml/home_assistant_streamdeck_yaml.py", line 1474, in get_deck
    streamdecks = DeviceManager().enumerate()
                  ^^^^^^^^^^^^^^^
  File "/Users/basnijholt/Code/python-elgato-streamdeck/src/StreamDeck/DeviceManager.py", line 92, in __init__
    self.transport = self._get_transport(transport)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/basnijholt/Code/python-elgato-streamdeck/src/StreamDeck/DeviceManager.py", line 84, in _get_transport
    raise ProbeError("Probe failed to find any functional HID backend.", probe_errors)
StreamDeck.DeviceManager.ProbeError: ('Probe failed to find any functional HID backend.', {'libusb': TransportError("No suitable LibUSB HIDAPI library found on this system. Is the 'libhidapi.dylib' library installed?")})
```

Could you tag a new version and release it? 😄 

Note to self: I can run it when setting:
```
export DYLD_LIBRARY_PATH=/opt/homebrew/lib/:$DYLD_LIBRARY_PATH
```

**Comment by abcminiuser** [comment]
Sure, sorry I've been a bit lax on the project the last few months - I've released 0.9.4.

**Comment by basnijholt** [comment]
Thanks! 😃

Happy user via https://github.com/basnijholt/home-assistant-streamdeck-yaml BTW
